Fails to build on PHP 7.4

If you try to build the image with PHP 7.4 it will fail to build with the following error:

```
configure: error: unrecognized options: --with-jpeg-dir, --with-png-dir
```

The options for this command have changed as per these [upgrade notes](https://github.com/php/php-src/blob/PHP-7.4/UPGRADING#L761-L770).

Should be pretty simple to fix. I'll write a PR.

--- a/publish/civicrm/php7.4/Dockerfile
+++ b/publish/civicrm/php7.4/Dockerfile
@@ -47,7 +47,7 @@ RUN curl -sL https://deb.nodesource.com/setup_10.x | bash - \
 # Install php extensions (curl, json, mbstring, openssl, posix, phar
 # are installed already and don't need to be specified here)
 RUN docker-php-ext-install bcmath \
-  && docker-php-ext-configure gd --with-jpeg-dir=/usr/include/ --with-png-dir=/usr/include/ \
+  && docker-php-ext-configure gd --with-jpeg \
   && docker-php-ext-install gd \
   && docker-php-ext-install gettext \
   && docker-php-ext-configure imap --with-kerberos --with-imap-ssl \

--- a/publish/templates/civicrm/Dockerfile.twig
+++ b/publish/templates/civicrm/Dockerfile.twig
@@ -51,7 +51,11 @@ RUN curl -sL https://deb.nodesource.com/setup_10.x | bash - \
 # Install php extensions (curl, json, mbstring, openssl, posix, phar
 # are installed already and don't need to be specified here)
 RUN docker-php-ext-install bcmath \
+{% if php_version in ['7.2', '7.3'] %}
   && docker-php-ext-configure gd --with-jpeg-dir=/usr/include/ --with-png-dir=/usr/include/ \
+{% else %}
+  && docker-php-ext-configure gd --with-jpeg \
+{% endif %}
   && docker-php-ext-install gd \
   && docker-php-ext-install gettext \
   && docker-php-ext-configure imap --with-kerberos --with-imap-ssl \

